packages/common/src/pipes/date_pipe_config.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Optionally provided date pipe configuration.
+ *
+ * @usageNotes
+ *
+ * Various date pipe default values can be overwritten by providing this interface.
+ *
+ * For example:
+ *
+ * Override the default date format by providing it in a module/component:
+ * ```typescript
+ * {provide: DATE_PIPE_DEFAULT_OPTIONS, useValue: {dateFormat: 'shortDate'}}
+ * ```
+ *
+ * Override the default timezone by providing it in a module/component:
+ * ```typescript
+ * {provide: DATE_PIPE_DEFAULT_OPTIONS, useValue: {timezone: '-1200'}}
+ * ```
+ *
```

Comment:
Let's drop the usage notes from here and point to the `DATE_PIPE_DEFAULT_OPTIONS` docs instead (so we don't have duplication of docs):
```suggestion
 * @see `DATE_PIPE_DEFAULT_OPTIONS`
```